packages/ses/src/whitelist.js

```diff
@@ -501,6 +501,9 @@ export const whitelist = {
     keyFor: fn,
     match: 'symbol',
     matchAll: 'symbol',
+    nodeAsyncHooksAsyncId: 'symbol',
+    nodeAsyncHooksTriggerAsyncId: 'symbol',
+    nodeAsyncHooksDestroyed: 'symbol',
```

Comment:
```suggestion
    // Non-standard, used in node to prevent async_hooks from breaking
    nodeAsyncHooksAsyncId: 'symbol',
    nodeAsyncHooksTriggerAsyncId: 'symbol',
    nodeAsyncHooksDestroyed: 'symbol',

```

Reply:
No longer relevant